### DISCUSSION TITLE
Go to File... based on git commit file history

### DISCUSSION THREAD

**Discussion by mattrybin** [microsoft/vscode-discussions#429] [Extension Development QnA] [answered]
<img width="843" alt="image" src="https://user-images.githubusercontent.com/7340953/215255087-d1f87349-2c78-498d-b6b4-bd665a11c511.png">

I use codespaces and I like that for every issue (A new codespace for each issue) that I work on, the "Go to File" panel is reset. However, I would like to be able to provide a custom list of files to the "Go to File" and I wonder where can I change the cache or whatever "Go to File" fetches from.

My goal is to fetch my git history and get the 15 most worked-on issues based on the feat, chore, or fix tags according to conventional commits standards.

- 15 most work on files for feat commits.
- 15 most work on files for fix commits.
- 15 most work on files for chore commits.

I don't need help with the git part, but I don't know how to update this "Go to File" manually. Any ideas?

**Comment by alefragnani** [comment, marked as answer]
Hi @mattrybin ,

AFAIK, there is no way to _provide a custom list_ or _manipulate the list_ of the `Go to File` command. 

You should instead, create your own command and populate it with the list of items you want.

Hope this helps
